### PR TITLE
NET Core: Update description for autoreferenced Packages

### PR DESCRIPTION
**Pull request by shishirx34**
Fixes https://github.com/NuGet/Home/issues/4964

The new description looks like[Edit: The new new description]:

image

### CODE REVIEW

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs [hunk 1]
```diff
@@ -510,7 +510,8 @@ protected void SetAutoReferencedCheck(bool autoReferenced)
             if (autoReferenced)
             {
                 OptionsBlockedUrl = new Uri("https://go.microsoft.com/fwlink/?linkid=841238");
-                OptionsBlockedMessage = "AutoReferenced";
+                var autoReferencedMessage = "This package is automatically referenced by the SDK and should not be updated independently. To update the package, update the SDK to which it belongs. More details:";
```
Comment (emgarten) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs]: This should be a resource string so that it can be localized.

**Review by emgarten — CHANGES_REQUESTED** [review]

**Comment by anangaur** [comment]
Looks so texty :( I need to refine the language to make it short.

**Review by anangaur — APPROVED** [review]
Looks good to me.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs [hunk 2]
```diff
@@ -449,6 +449,24 @@ public virtual Uri OptionsBlockedUrl
...
+            {
+                return _optionsBlockedUrlText;
+            }
+
+            set
+            {
+                _optionsBlockedUrlText = value;
+                OnPropertyChanged(nameof(OptionsBlockedUrlText));
+                OnPropertyChanged(nameof(OptionsBlockedUrl));
+                OnPropertyChanged(nameof(OptionsBlockedMessage));
```
Comment (shishirx34) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs:466]: Wonder if these other property changed events need to be raised here. Mimicked the behavior from above to get feedback.

**Review by emgarten — APPROVED** [review]

**Review by jainaashish — APPROVED** [review]

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs [hunk 3]
```diff
@@ -510,8 +528,8 @@ protected void SetAutoReferencedCheck(bool autoReferenced)
             if (autoReferenced)
             {
                 OptionsBlockedUrl = new Uri("https://go.microsoft.com/fwlink/?linkid=841238");
-                OptionsBlockedMessage = "AutoReferenced";
-
+                OptionsBlockedMessage = Resources.AutoReferenced;
+                OptionsBlockedUrlText = "Learn more";
```
Comment (alpaix) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs]: the text should be localized

Comment (shishirx34) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs]: I assumed "Learn more" need not be localized. Updated.

**Review by alpaix — APPROVED** [review]

**Comment by laultman** [comment]
From a everyday user: Lots of discussion that doesn't help me actually fix my issue. I can very quickly see that I need to upgrade something. I look in my Program Files and see that I have the latest SDK for .NET Core. installed. In fact SDK 2.0.0 is NOT installed. I have 2.1.2. It still will not upgrade in my solution. Why don't you address that?

**Pull request merged**